modules/blosc.h

```diff
@@ -0,0 +1,98 @@
+// # BlOsc
+//
+// Band Limited Oscillator
+//
+// ## Credits  
+// * Based on bltriangle, blsaw, blsquare from soundpipe
+// * Original Author(s): Paul Batchelor, saw2 Faust by Julius Smith
+// * Ported by Ben Sergentanis, May 2020
+
+#pragma once
+#ifndef DSY_BLOSC_H
+#define DSY_BLOSC_H
+
+#include <stdint.h>
+#ifdef __cplusplus
+
+namespace daisysp
+{
+    // Bl Waveforms
+    // ~~~~
+    enum
+    {
+        BL_WAVEFORM_TRIANGLE,
+        BL_WAVEFORM_SAW,
+        BL_WAVEFORM_SQUARE,
+    };
+    // ~~~~
```

Comment:
name the the type, (i.e. `enum Waveforms`) and move it inside the class.  (You can change the long-name notation at that point to `WAVE_NAME` since it will be accessed as `BlOsc::WAVE_TRIANGLE`, etc.